build/ci/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -xe
+set +o pipefail
+
+python -m pip install --upgrade pip docker-hub==2.2.0
+echo yq > dev-requirements.txt
+
+cat >>/home/runner/.bash_profile <<'EOL'
+yq() {
+  docker run --rm -e operator_image_for_test=$operator_image_for_test\
+                  -e controller_repository_for_test=$controller_repository_for_test\
+                  -e node_repository_for_test=$node_repository_for_test\
+                  -e driver_images_tag=$driver_images_tag\
+                  -i -v "${PWD}":/workdir mikefarah/yq "$@"
+}
+EOL
+
+source /home/runner/.bash_profile
+cd deploy/olm-catalog/ibm-block-csi-operator
```

Comment:
what about community?

Reply:
I only need to take the latest version of our product

Reply:
we don't bump the operator version and the `currentCSV` in the same PR, and regardless, this script shouldn't rely on the existence of our support for olm (unless it's specifically for testing with olm).

could you take it from [here](https://github.com/IBM/ibm-block-csi-operator/blob/d4f17ed8cf496ebd4f9a600506cfb2164aa878c1/version/version.go#L20) instead, and open a ticket to move this value into a common config file?

Reply:
could you write the ticket number in this conversation?

Reply:
CSI-3173